src/Directory.Build.props

```diff
@@ -1,8 +1,24 @@
 <Project>
     <PropertyGroup>
+        <LangVersion>8.0</LangVersion>
+        <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+
+        <Title>GitReleaseManager Tool</Title>
```

Comment:
I am not sure about making this one a common property. That means that each assembly would be stamped with the same name, rather than the unique title that it is currently getting. For example, I think the GitReleaseManager.Core be stamped with that title.

Reply:
I think Title can be kept, same for Description, I will update